File: Simplace/SimplaceReferenceETPriestleyTaylor/src/cs/SimplaceReferenceETPriestleyTaylor/ReferenceETPriestleyTaylor_Auxiliary.cs
using System;
using System.Collections.Generic;

public class ReferenceETPriestleyTaylor_Auxiliary 
{
    private double _ReferenceCropEvapotranspiration;
    
    /// <summary>
    /// Constructor of the ReferenceETPriestleyTaylor_Auxiliary component")
    /// </summary>  
    public ReferenceETPriestleyTaylor_Auxiliary() { }
    
    
    public ReferenceETPriestleyTaylor_Auxiliary(ReferenceETPriestleyTaylor_Auxiliary toCopy, bool copyAll) // copy constructor 
    {
        if (copyAll)
        {
    
            ReferenceCropEvapotranspiration = toCopy.ReferenceCropEvapotranspiration;
        }
    }
    public double ReferenceCropEvapotranspiration
    {
        get { return this._ReferenceCropEvapotranspiration; }
        set { this._ReferenceCropEvapotranspiration= value; } 
    }
}

File: Simplace/SimplaceReferenceETPriestleyTaylor/src/java/SimplaceReferenceETPriestleyTaylor/ReferenceETPriestleyTaylor_Auxiliary.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class ReferenceETPriestleyTaylor_Auxiliary
{
    private double ReferenceCropEvapotranspiration;
    
    public ReferenceETPriestleyTaylor_Auxiliary() { }
    
    public ReferenceETPriestleyTaylor_Auxiliary(ReferenceETPriestleyTaylor_Auxiliary toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.ReferenceCropEvapotranspiration = toCopy.getReferenceCropEvapotranspiration();
        }
    }
    public double getReferenceCropEvapotranspiration()
    { return ReferenceCropEvapotranspiration; }

    public void setReferenceCropEvapotranspiration(double _ReferenceCropEvapotranspiration)
    { this.ReferenceCropEvapotranspiration= _ReferenceCropEvapotranspiration; } 
    
}